src/components/__stories__/login_header.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react'
+import { storiesOf, action } from "@kadira/storybook"
+
+import LoginHeader from "../login_header"
+
+storiesOf("LoginHeader", LoginHeader)
+  .add("Simple login header", () => (
+    <LoginHeader/>
+  ))
```

Comment:
Do you think there will be multiple header stories? If not, maybe we can make this a ‘static/simple’ components group?